pkg/metrics/service.go

```diff
@@ -71,8 +71,15 @@ func New(ctx context.Context, idCfg *config.IdentityConfig) (daemon.Daemon, erro
 		ConfigFile:    "",
 		Files: func() []string {
 			files := []string{}
-			if idCfg.CertFile != "" {
-				files = append(files, idCfg.CertFile)
+			if idCfg.ServiceCert.CopperArgos.Use {
+				for _, certFile := range idCfg.ServiceCert.CopperArgos.Cert.Paths {
+					if certFile != "" {
+						files = append(files, certFile)
+					}
```

Comment:
With `splitAndFilterPaths()`, no need:
```suggestion
					files = append(files, certFile)
```

Reply:
fixed in https://github.com/AthenZ/k8s-athenz-sia/pull/192/commits/3aa34307ea4042f642a86ff2c43f9a11620a0398